2016/cities.html

```diff
@@ -0,0 +1,147 @@
+          <span class="stat__number">207,770</span>
+        </p>
+        <p class="stat__comment">
+          FixMyStreet reports, sent to 421 UK councils and transport bodies in 2016. That’s up almost 25% on last year.
+        </p>
+      </div>
+    </div>
+
+  </div>
+  <div class="row">
+
+    <div class="column-one-half">
+      <div class="stat">
+        <p class="stat__main">
+          <span class="stat__number">1,195</span>
+        </p>
+        <p class="stat__comment">
+          Reports of cycling accidents and near misses on <a href="https://www.mysociety.org/better-cities/case-study-collideoscope/">Collideoscope</a>.
+        </p>
+      </div>
+    </div>
+
+    <div class="column-one-half"></div>
+
+  </div>
+</div>
+
+<div class="section section--centred section--cities-tweets">
+  <h1>In 2016, FixMyStreet.com helped people like…</h1>
+
+  <div class="row">
+    <div class="column-one-half">
+      {% include tweet.html user_display_name="Dan Stowell" user_handle="mclduk" tweet_html='Mattress lying in the road for a month. Reported it using <a href="https://twitter.com/fixmystreet">@FixMyStreet</a> -it’s gone the next day. Thanks <a href="https://twitter.com/TowerHamletsNow">@TowerHamletsNow</a>, thanks <a href="https://twitter.com/fixmystreet">@FixMyStreet</a>' tweet_url="https://twitter.com/mclduk/status/760844400456970240" date="3 Aug 2016"  %}
+    </div>
+    <div class="column-one-half">
+      {% include tweet.html user_display_name="Tim Cowland" user_handle="Tim_SBIG" tweet_html='Amazing service from <a href="https://twitter.com/Herts_Highways">@Herts_Highways</a>. Faulty street lamp reported via <a href="https://twitter.com/fixmystreet">@fixmystreet</a> fixed with 34 mins of sending message. <a href="https://twitter.com/hashtag/speedoflight?src=hash">#speedoflight</a>' tweet_url="https://twitter.com/Tim_SBIG/status/725256139051536384" date="27 Apr 2016"  %}
+    </div>
+  </div>
+  <div class="row">
+    <div class="column-one-half">
+      {% include tweet.html user_display_name="Eadlin" user_handle="DeeHants" tweet_html='<a href="https://twitter.com/mysociety">@mysociety</a>, your <a href="https://twitter.com/fixmystreet">@fixmystreet</a> app makes it so much easier to report issues. I found another 3 on today’s drive home!' tweet_url="https://twitter.com/DeeHants/status/722131033253089280" date="18 Apr 2016"  %}
+    </div>
+    <div class="column-one-half">
+      {% include tweet.html user_display_name="Plexus Design" user_handle="plexusdesigner" tweet_html='<a href="https://fixmystreet.com">fixmystreet.com</a> 3 min form has fixed an issue that has been bugging me and my neighbours for months! Thank you! <a href="https://twitter.com/mysociety">@mysociety</a>' tweet_url="https://twitter.com/plexusdesigner/status/709519365603991552" date="14 Mar 2016"  %}
+    </div>
+  </div>
+</div>
+
+<div class="section section--white section--cities-councils">
+  <h1>Huge improvements on FixMyStreet for Councils…</h1>
```

Comment:
I think you already commented on this but I can't find it now! But yes, as Paul suggested, let's flag up that this is commercial a bit more clearly.

> FixMyStreet powers several street fault reporting systems for councils in the UK and beyond, as a <a href="http://www.fixmystreet.com/councils">commercial service</a> that gives them more flexibility, slashed costs and high usability.

> This year, the service has been totally transformed for some of our most important users: the council staff who receive reports. It’s all thanks to our collaboration with some of the UK’s most forward-thinking authorities.